control-plane/plugin/src/resources/volume.rs

```diff
@@ -225,6 +227,63 @@ impl Scale for Volume {
     }
 }
 
+#[async_trait(?Send)]
+impl SetProperty for Volume {
+    type ID = VolumeId;
+    async fn set_property(
+        id: &Self::ID,
+        property: &str,
+        output: &utils::OutputFormat,
+    ) -> PluginResult {
+        let parts: Vec<&str> = property.trim().split('=').collect();
+        if parts.len() != 2 {
+            return Err(Error::SetVolumePropertyParseError {
+                reason: format!("volume: {id}, invalid property format: {property}, enter in the format property=value"),
+            });
+        }
+        let left = parts[0].trim();
+        let right = parts[1].trim();
+        let property_body = match left {
```

Comment:
Unrelated to this PR. @tiagolobocastro Just wondering if we should use strum on generated code to get these conversions?

Reply:
I think we should use explicit types to begin with, perhaps even use the openapi properties in the CLI